src/bindings/python/tests/test_graph/test_random_uniform.py

```diff
@@ -19,3 +23,125 @@ def test_random_uniform():
     assert random_uniform_node.get_type_name() == "RandomUniform"
     assert random_uniform_node.get_output_element_type(0) == Type.f32
     assert list(random_uniform_node.get_output_shape(0)) == [2, 4, 3]
+
+
+@pytest.mark.parametrize(
+    ("min_value", "max_value"),
+    [
+        (-10, 10),
+        (-100, 123)
+    ],
+)
+@pytest.mark.parametrize(
+    ("global_seed", "op_seed"),
+    [
+        (1, 2),
+        (367, 123),
+        (93, 43)
+    ]
+)
+@pytest.mark.parametrize(
+    ("ov_dtype", "torch_dtype", "str_dtype"),
+    [
+        (Type.f64, torch.float64, "f64"),
+        (Type.f32, torch.float32, "f32"),
+        (Type.f16, torch.float16, "f16"),
+        (Type.bf16, torch.bfloat16, "bf16"),
+        (Type.i64, torch.int64, "i64"),
+        (Type.i32, torch.int32, "i32")
+    ]
+)
+@pytest.mark.parametrize(
+    ("shape"),
+    [
+        (4, 4),
+        (6, 6),
+        (10, 5)
+    ]
+)
+def test_random_uniform_pytorch_alignment(min_value, max_value, global_seed, op_seed, ov_dtype, torch_dtype, str_dtype, shape):
+
+    # PyTorch does not have a dedicated RandomUniform op per-se,
+    # the closest one is either torch.rand / torch.randint,
+    # this implementation internally uses the same computation path
+    # without having to select between float and int data type.
+    torch.manual_seed(global_seed)
+    expected = torch.empty(shape, dtype=torch_dtype)
+    expected = expected.uniform_(min_value, max_value)
+
+    core = Core()
+    minval, maxval = ops.constant(min_value, ov_dtype), ops.constant(max_value, ov_dtype)
+    output = ops.random_uniform(Shape(shape), minval, maxval, str_dtype, global_seed, op_seed, "pytorch")
+    model = Model(output, [], "random_uniform_seed_alignment_test_model_pytorch")
+    compiled_model = core.compile_model(model, "TEMPLATE")
+    results = compiled_model.infer_new_request()
+    actual = next(iter(results.values()))
+
+    assert np.array_equal(expected, actual)
+
+
+@pytest.mark.parametrize(
+    ("min_value", "max_value"),
+    [
+        (-10, 10),
+        (-100, 123)
+    ],
+)
+@pytest.mark.parametrize(
+    ("global_seed", "op_seed"),
+    [
+        (1, 2),
+        (367, 123),
+        (93, 43)
+    ]
+)
+@pytest.mark.parametrize(
+    ("ov_dtype", "tensorflow_dtype", "str_dtype"),
+    [
+        (Type.f64, tf.float64, "f64"),
+        (Type.f32, tf.float32, "f32"),
+        (Type.f16, tf.float16, "f16"),
+        (Type.bf16, tf.bfloat16, "bf16"),
+        (Type.i64, tf.int64, "i64"),
+        (Type.i32, tf.int32, "i32")
+    ]
+)
+@pytest.mark.parametrize(
+    ("shape"),
+    [
+        (4, 4),
+        (6, 6),
+        (10, 5)
+    ]
+)
+def test_random_uniform_tensorflow_alignment(min_value, max_value, global_seed, op_seed, ov_dtype, tensorflow_dtype, str_dtype, shape):
```

Comment:
This directory is for python binding tests, 
comparison with framework results should follow the corresponding frontend tests (with frontend op update to use specific mode).

Example:
https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/tensorflow_tests/test_tf_RandomUniform.py
or 
https://github.com/openvinotoolkit/openvino/tree/master/tests/layer_tests/pytorch_tests


At this point specific tests of the new attribute with hardcoded expected values should be added to the cpp reference tests:
https://github.com/openvinotoolkit/openvino/blob/17f8e86e5f29cddaba0e6ef62f006639c56155c3/src/plugins/template/tests/functional/op_reference/random_uniform.cpp#L91-L93

Reply:
Done!